airflow/ui/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
```

Comment:
I think that most of that is handled by the root gitignore. I believe this was generated automatically by the vite template but we can safely remove it because there is a lot of overlap. (Maybe just the pnpm-debug.log* and a couple others are missing)

Reply:
Ok. Consolidated